### PR TITLE
fix(benchmark): Benchmark improvements

### PR DESCRIPTION
**Pull request by tomi**
## Summary

- chore: Pin nginx image version
- refactor: Log nginx errors to stderr
- refactor: Log if invalid response is received in benchmarks
- refactor: Remove VM disk benchmark 

The disk performance is pretty much stable and there isn't much variance

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/@n8n/benchmark/scenarios/js-code-node/js-code-node.script.js [hunk 1]
```diff
@@ -5,6 +5,13 @@ const apiBaseUrl = __ENV.API_BASE_URL;
 
 export default function () {
 	const res = http.post(`${apiBaseUrl}/webhook/code-node-benchmark`, {});
+
+	if (res?.status !== 200) {
```
Comment (ivov) [inline packages/@n8n/benchmark/scenarios/js-code-node/js-code-node.script.js:9]: From what I can tell, response is always defined - it's the response body that can be `undefined`. Same for others.

```ts
/**
 * Refined response.
 * Exposes body with best possible type.
 */
export interface RefinedResponse<RT extends ResponseType | undefined> extends Response {
    body: RefinedResponseBody<RT>;
}
```

**Review by ivov — COMMENTED** [review]
When debugging locally I found that the binary files scenario in multi-main was _not_ failing, if it helps as a clue.

File: packages/@n8n/benchmark/scenarios/js-code-node/js-code-node.script.js (hunk 1 above)
Comment (tomi) [reply packages/@n8n/benchmark/scenarios/js-code-node/js-code-node.script.js:9]: Alright, I'll remove the question marks

**Comment by tomi** [comment]
> When debugging locally I found that the binary files scenario in multi-main was _not_ failing, if it helps as a clue.

Yeah I haven't been able to reproduce this either locally or when running manually in the CI. Only once I managed to 2 failed requests for the HTTP request node benchmark. That's why it's a bit painful to debug

**Review by ivov — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.65.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.65.0)

**Pull request merged**